src/github/client/github-client.ts

```diff
@@ -99,10 +99,24 @@ export default class GitHubClient {
 		return response;
 	}
 
```

Comment:
We've decide it to add it to the new GitHub Client, and use the new client to make the call. (To not do the work twice)